File: app/src/main/java/com/babestudios/companyinfouk/data/model/officers/Officers.java
package com.babestudios.companyinfouk.data.model.officers;


import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Officers {
	@SerializedName("kind")
	public String kind;
	@SerializedName("items_per_page")
	public int itemsPerPage;
	@SerializedName("total_results")
	public int totalResults;
	@SerializedName("active_count")
	public int activeCount;
	@SerializedName("start_index")
	public int startIndex;
	@SerializedName("etag")
	public String etag;
	@SerializedName("items")
	public List<OfficerItem> items = new ArrayList<>();
	@SerializedName("resigned_count")
	public int resignedCount;
	@SerializedName("links")
	public OfficersLinks links;

}

File: data/src/main/java/com/babestudios/companyinfouk/data/model/officers/Officers.kt
package com.babestudios.companyinfouk.data.model.officers


import com.babestudios.companyinfouk.data.model.common.SelfLinkData
import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class Officers (
		@SerializedName("kind")
	var kind: String? = null,
		@SerializedName("items_per_page")
	var itemsPerPage: Int = 0,
		@SerializedName("total_results")
	var totalResults: Int = 0,
		@SerializedName("active_count")
	var activeCount: Int = 0,
		@SerializedName("start_index")
	var startIndex: Int = 0,
		@SerializedName("etag")
	var etag: String? = null,
		@SerializedName("items")
	var items: List<OfficerItem> = ArrayList(),
		@SerializedName("resigned_count")
	var resignedCount: Int = 0,
		@SerializedName("links")
	var links: SelfLinkData? = null

)
